lib/escape.js

```diff
@@ -53,6 +53,20 @@ exports.escapeHtml = function (input) {
 };
 
 
+exports.escapeJson = function (input) {
+
+    if (!input) {
+        return '';
+    }
+
+    return input.replace(/</g, '\\u003c')
```

Comment:
The current implementation here seems like it could be made quite a bit more performant by reducing the number of calls to `String.replace(...)` which would search the entire string again.

Something along these lines would be more performant:

```js

return input.replace(/[<>&\u2028\u2029]/g, (match) => {
  if (match === '<') {
    return '\\u003c';
  } else if (...) {
    ...
  }
  // etc
});